### PR TITLE
qa: test mgr plugins with failover

### PR DESCRIPTION
**Pull request by mchangir**
Following mgr plugins are being tested:
* snap_schedule
* mirroring

Fixes: https://tracker.ceph.com/issues/56489
Signed-off-by: Milind Changire <mchangir@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [x] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: qa/suites/fs/mgr-failover/mgr-failover.yaml [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  - exec:
+      mon.a:
+        # thrash mgr
+        - sleeps=( 60 30 30 30 30 60 30 30 30 60 120 );
+          sleep 300;
+          for i in ${sleeps[*]}; do
+            sleep $i;
+            active_mgr=$(ceph mgr dump | jq --raw-output .active_name);
+            echo "killing active mgr '$active_mgr'";
+            ceph mgr fail $active_mgr;
```
Comment (vshankar) [inline qa/suites/fs/mgr-failover/mgr-failover.yaml:70]: Any reason to not have this as a thrasher task?

Comment (mchangir) [reply qa/suites/fs/mgr-failover/mgr-failover.yaml:70]: - there wasn't anything significantly sophisticated in the thrasher that I wrote up
- there was a single file that contained everything needed to address the effort

Comment (vshankar) [reply qa/suites/fs/mgr-failover/mgr-failover.yaml:70]: What do you mean? If you look at mds_thrash.py, there are a handful of tunables that are used to thrash the mds if different ways/time-points. Not all of those tunables are valid to ceph-mgr, however, I think a thrasher task is much suited for this.

**Comment by vshankar** [comment]
@mchangir ping?

**Comment by mchangir** [comment]
> @mchangir ping?

haven't looked at this in a while
tied up with an mds crash issue

**Pull request closed without merging**